terraform/eks-node-groups/multi-az-node-groups/launch_templates.tf

```diff
@@ -0,0 +1,59 @@
+
+
+module "launch_templates" {
+  source   = "github.com/pluralsh/module-library//terraform/eks-node-groups/launch-template?ref=feat-ubuntu-ng"
```

Comment:
Since the node group and launch template modules are being bundled together (and will be under the same git version tag), wouldn't it be possible and better to use a local reference here?

Reply:
did this now, we pass the launch templates separately in the sysbox artifact now, as an example
this removes the nested dependency, this way it's more modular